packages/react-components/src/form.ts

```diff
@@ -83,21 +88,25 @@ export function useValidation<T extends ValidationTarget>(
     validationMessage,
     validationTargetProps: {
       ref: inputRef,
-      onBlur: (event: FocusEvent<ValidationTarget>) => {
+      onBlur: () => {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Might be worth the comment or putting in the handle gracefully and ignore coverage.

Reply:
Tbh I'd like it to throw if for some reason this is null. It's a path that should be impossible to produce. If it doesn't throw, it's harder to figure out why validation isn't working